### DISCUSSION TITLE
Export issues

### DISCUSSION THREAD

**Discussion by aplasa85** [neuefische/questions#396] [Web] [answered]
I have been trying to implement my product page with the newly merged product list. Now, it seems as if I am not allowed to a. have anything with styled components, as it is not recognised anywhere, b, to have an export default declaration. If I ommit it then it does not render to the browser as it is not defined. When I put it I am told I "cannot have multiple default export "declarations. In that case I get an empty page with no element at all, and any addition to that is solely a source of new errors. Any hint on what is happening here?

<img width="1176" alt="image" src="https://user-images.githubusercontent.com/107868311/207638041-b37bf1ae-eaf0-4f59-9923-bd70114f3271.png">

https://github.com/aplasa85/capstone-template-ffm-web-22-1

**Comment by ahohnsen** [comment, marked as answer]
Hey there,

Folder and file names in your `pages` folder should be always in lowercase as pages are associated with a route based on their file name. So the folder `Products` needs to be renamed to `products` and `[ProductId]` to `[productId]`. Then you should be able to reach your product page.

Additionally, the file `StyledCard` has no file extension and therefore the import is not working. As you are using styled-components the file extension needs to be `*.js` -> `StyledCard.js`. 🙂 

☝🏼  **For the next time:** Please, keep in mind to mention in your post in which branch you are currently working, so it is easier to find the bit of code related to your question. 🙏🏼
